### PR TITLE
dev-games/aseprite: updates to package

### PR DESCRIPTION
**Pull request by winterheart**
Update version to 1.1.9 which now uses Aseprite-EULA license (here discussion about approval: https://archives.gentoo.org/gentoo-licenses/message/4f144e6c91c8391917ba66024f5ae8ee).
Also fixed bug #660534 and migrated to EAPI 7 and cmake eclass.

### CODE REVIEW

File: dev-games/aseprite/aseprite-1.1.9.ebuild [hunk 1]
```diff
@@ -0,0 +1,109 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake desktop eutils flag-o-matic
+
+DESCRIPTION="Animated sprite editor & pixel art tool"
+HOMEPAGE="http://www.aseprite.org"
```
Comment (juippis) [inline dev-games/aseprite/aseprite-1.1.9.ebuild]: `https://` is available.

File: dev-games/aseprite/aseprite-1.1.9.ebuild [hunk 2]
```diff
@@ -0,0 +1,109 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake desktop eutils flag-o-matic
```
Comment (juippis) [inline dev-games/aseprite/aseprite-1.1.9.ebuild]: Hmm, I didn't see *eutils* being used.

File: dev-games/aseprite/aseprite-1.1.9.ebuild [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+	media-libs/freetype
+	media-libs/giflib:=
+	webp? ( media-libs/libwebp )
+	media-libs/libpng:0=
+	net-misc/curl
+	sys-libs/zlib
+	virtual/jpeg:0
+	x11-libs/libX11
+	x11-libs/pixman
+	gtk3? ( dev-cpp/gtkmm:3.0 )
```
Comment (juippis) [inline dev-games/aseprite/aseprite-1.1.9.ebuild]: We use styling where we sort unconditional deps before conditional ones.

File: dev-games/aseprite/aseprite-1.1.9.ebuild [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+	kde? (
+		dev-qt/qtcore:5
+		kde-frameworks/kio:5 )"
```
Comment (juippis) [inline dev-games/aseprite/aseprite-1.1.9.ebuild]: You should also multi-line properly.

File: dev-games/aseprite/aseprite-1.1.9.ebuild [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+DEPEND="${RDEPEND}
+	app-arch/unzip
+	gtk3? ( virtual/pkgconfig )
+	webp? ( virtual/pkgconfig )"
```
Comment (juippis) [inline dev-games/aseprite/aseprite-1.1.9.ebuild]: With EAPI-7 these are all BDEPEND.

**Comment by winterheart** [comment]
@juippis done.

**Pull request closed without merging**